src/integration/prepareApiReferences.js

```diff
@@ -49,15 +49,17 @@ function prepareApiReferences(registry, config, next) {
 
         // create API reference file
         createApiReferenceMainFile(apiReferenceSource, apiReferenceNewFile, apiReferenceContent, name),
-        createApiReferenceMainFile(apiReferenceSourceInternal, apiReferenceNewFileInternal, apiReferenceContentInternal, nameInternal),
+        createApiReferenceMainFile(apiReferenceSourceInternal, apiReferenceNewFileInternal, apiReferenceContentInternal, nameInternal)
 
+        // *** temporarily disabled ***
```

Comment:
`temporarily disabled` doesn't say much, in case we want to use this feature again we want to find it quickly in code, IMHO something like

```
/* <CLIENT.JS GENERATION>
// commented code
// commented code
  </CLIENT.JS GENERATION> */

so we can see EXACTLY what we have to comment out to make this feature work.

Ofc HTML braces are just an example, you can try some other method of commenting it